lib/matplotlib/axes/_axes.py

```diff
@@ -1674,7 +1674,9 @@ def xcorr(self, x, y, normed=True, detrend=mlab.detrend_none,
         """
         Plot the cross correlation between *x* and *y*.
 
-        The correlation with lag k is defined as sum_n x[n+k] * conj(y[n]).
+        The correlation with lag k is defined as
+        :math:`\sum_n x[n+k] \cdot y^*[n]`, where :math:`y^*` is the complex
```

Comment:
OK, I can't run the tests anymore because of this math call.  (SyntaxError: invalid escape sequence \s).  What do I do to not get this to happen?

Reply:
my bad, see https://github.com/matplotlib/matplotlib/pull/10877